platform-ajax/src/test/java/com/softicar/platform/ajax/dom/event/click/AjaxDomClickEventTest.java

```diff
@@ -65,6 +67,9 @@ public void testWithScrolledWindow() {
 		AjaxSeleniumTestSegment backgroundDivSize = getSize(backgroundDiv);
 		setViewportSize(backgroundDivSize.getWidth() - SCROLL_X, backgroundDivSize.getHeight() - SCROLL_Y);
 
+		// race condition: resizing apparently takes some time
+		Sleep.sleep(Duration.ofMillis(500));
```

Comment:
that is nasty, solutions?

Reply:
Changed `AjaxSeleniumLowLevelTestEngineViewport.setViewportSize` to use a `WebDriverWait` with a timeout of 10s instead. Pls check it out.

Reply:
better, see comment